drivers/iommu/arm/arm-smmu/arm-smmu-qcom.c

```diff
@@ -446,6 +446,7 @@ static const struct of_device_id __maybe_unused qcom_smmu_impl_of_match[] = {
 static struct acpi_platform_list qcom_acpi_platlist[] = {
 	{ "LENOVO", "CB-01   ", 0x8180, ACPI_SIG_IORT, equal, "QCOM SMMU" },
 	{ "QCOM  ", "QCOMEDK2", 0x8180, ACPI_SIG_IORT, equal, "QCOM SMMU" },
+	{ "QCOM  ", "QCOMEDK2", 0x8280, ACPI_SIG_IORT, equal, "QCOM SMMU" },
```

Comment:
Are you sure that `QCOM`, `QCOMEDK2` is the right combination here? AFAIK that's for Qualcomm's developer kit. Although it could be possible that MS is re-using that.

Reply:
Here's a photo of the ACPI logs from boot attempt: If you've got instructions on how to get a dump out of windows /wsl I can do that too 
![acpi_dump](https://user-images.githubusercontent.com/2041026/201063036-81ec4c26-095d-405c-96d7-ec21fce11184.jpg)

Reply:
`acpidump` and other ACPICA tools are actually also available for windows: https://acpica.org/downloads/binary-tools. So you can use that directly from Windows. However, before posting don't forget to filter out the MSDM section containing some Windows licensing information (e.g. as described here: https://github.com/linux-surface/linux-surface/wiki/Testers-and-Contributing#contributing-acpidumps.